interface/billing/billing_tracker.php

```diff
@@ -0,0 +1,188 @@
+    <title><?php echo xlt("Claim File Tracker"); ?></title>
+    <style>
+        table.dataTable td.details-control:before {
+            content: '\f152';
+            font-family: 'Font Awesome\ 5 Free';
+            cursor: pointer;
+            font-size: 22px;
+            color: #55a4be;
+        }
+        table.dataTable tr.shown td.details-control:before {
+            content: '\f150';
+            color: black;
+        }
+    </style>
+    <script type="text/javascript">
+        $(document).ready(function() {
+            const serverUrl = "<?php echo $GLOBALS['webroot']; ?>/library/ajax/billing_tracker_ajax.php?csrf_token_form=" + <?php echo js_url(CsrfUtils::collectCsrfToken()); ?>;
+            const oTable = $('#billing-tracker-table').DataTable({
+                "processing": true,
+                // next 2 lines invoke server side processing
+                "ajax": {
+                    "type" : "GET",
+                    "url" : serverUrl,
+                    "dataSrc": function (json) {
+                        return json.data;
+                    }
+                },
+                "columns": [
+                    {
+                        "class": 'details-control',
+                        "orderable": false,
+                        "data": null,
+                        "defaultContent": ''
+                    },
+                    {
+                        "data": "status",
+                        "render": function(data, type, row, meta) {
+                            // Format the status with a nice looking badge
+                            if (type === 'display') {
+                                if (data == 'success') {
+                                    data = '<span class="badge badge-success">' + jsText(data) + '</span>';
+                                } else if (data == 'waiting') {
+                                    data = '<span class="badge badge-info">' + jsText(data) + '</span>';
+                                } else {
+                                    data = '<span class="badge badge-warning">' + jsText(data) + '</span>';
+                                }
+                            }
+
+                            return data;
+                        }
+                    },
+                    { "data": "x12_partner_name" },
+                    {
+                        "data": "x12_filename",
+                        "render": function(data, type, row, meta) {
+                            // Build the URL so the user can download the claim batch file
+                            if (type === 'display') {
+                                const url = '<?php echo $GLOBALS['webroot']; ?>/interface/billing/get_claim_file.php?' +
+                                    'key=' + encodeURIComponent(data) +
+                                    '&csrf_token_form=<?php echo js_url(CsrfUtils::collectCsrfToken()); ?>' +
```

Comment:
```
'&csrf_token_form=' + <?php echo js_url(CsrfUtils::collectCsrfToken()); ?> +
```
(the js part of this function uses js_encode and will place quotes around it, so need to separate it out)